### PR TITLE
Task 208 implement cucumber steps for multi drop down element

### PR DESCRIPTION
**Pull request by hellokitties**
BDD for MultiDropDown (which is MultiSelector)

### CODE REVIEW

File: jdi-light-bdd/src/main/java/com/epam/jdi/bdd/stepdefs/MultiDropDownSteps.java [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+        multiDropdown.check(value);
+    }
+
+    @Then("^the \"([^\"]*)\" selected value is \"([^\"]*)\"$")
+    public void theSelectedValueIs(String name, String value) {
+        MultiSelector multiDropdown = getUI(name, MultiSelector.class);
+        assertEquals(multiDropdown.selected(), value);
+    }
+
+    @When("^(?:I |)select in the \"([^\"]*)\" value \"([^\"]*)\"$")
```
Comment (ChepelVE) [inline jdi-light-bdd/src/main/java/com/epam/jdi/bdd/stepdefs/MultiDropDownSteps.java]: @When("^(?:I |)select in the \"([^\"]*)\" value \"([^\"]*)\"$") already exist in JDISteps

File: jdi-light-bdd-tests/src/main/java/io/github/com/enums/Ages.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package io.github.com.enums;
```
Comment (ChepelVE) [inline jdi-light-bdd-tests/src/main/java/io/github/com/enums/Ages.java]: We don't need this enum to be declared.

File: jdi-light-bdd-tests/src/test/java/cucmberTests/Runner.java [hunk 3]
```diff
@@ -26,7 +26,7 @@
 @CucumberOptions(
         features = "classpath:features"
         , glue = {"com.epam.jdi.bdd", "cucmberTests"}
-        //, tags = {"@menu"}
+        , tags = {"@MultiDropDown"}
```
Comment (ChepelVE) [inline jdi-light-bdd-tests/src/test/java/cucmberTests/Runner.java]: should be commented in bdd branch, now it's only for local purposes

File: jdi-light-bdd-tests/src/test/resources/features/MultiDropDown.feature [hunk 4]
```diff
@@ -0,0 +1,54 @@
...
+      | Disabled | Wood | Steam | Electro | Metalic |
+
+  Scenario: Check disabled element value
+    When I select in the "Multi Dropdown" value "Disabled"
+    Then the "Multi Dropdown" selected value is "Steam"
+
+  Scenario: Validation test
+    Given I open "Html5 Page"
+    Then the "Multi Dropdown" selected value is "Steam"
+    Then the "Multi Dropdown" values has item "Wood"
```
Comment (ChepelVE) [inline jdi-light-bdd-tests/src/test/resources/features/MultiDropDown.feature]: use And

File: jdi-light-bdd/src/main/java/com/epam/jdi/bdd/stepdefs/MultiDropDownSteps.java (hunk 1 above)
Comment (hellokitties) [reply jdi-light-bdd/src/main/java/com/epam/jdi/bdd/stepdefs/MultiDropDownSteps.java]: fixed

File: jdi-light-bdd-tests/src/test/java/cucmberTests/Runner.java (hunk 3 above)
Comment (hellokitties) [reply jdi-light-bdd-tests/src/test/java/cucmberTests/Runner.java]: fixed

File: jdi-light-bdd-tests/src/test/resources/features/MultiDropDown.feature (hunk 4 above)
Comment (hellokitties) [reply jdi-light-bdd-tests/src/test/resources/features/MultiDropDown.feature]: fixed

**Pull request merged**